pgcli/main.py

```diff
@@ -1624,6 +1625,8 @@ def cli(
     else:
         pgcli.connect(database, host, user, port)
 
+    pgcli.pgexecute.set_timezone(tzlocal.get_localzone_name())
```

Comment:
Does this ever raise an exception or return a None?
What happens if I set `TZ` to a non-existent timezone name?